content/resources/gen3-features.md

```diff
@@ -33,6 +35,9 @@ Gen3 is an open-source platform that enables the standing-up of data commons for
 * Import and export using the Portable Format for Biomedical Data (PFB): *(Work in progress)*
     * Import and export a dictionary
     * Import and export structured data
+* Setup [Metadata Services (MDS)](https://github.com/uc-cdis/metadata-service#metadata-service) and programmatically access semi-structured metadata using the [endpoint](https://petstore.swagger.io/?url=https://raw.githubusercontent.com/uc-cdis/metadata-service/master/docs/openapi.yaml)
```

Comment:
How about:

[Enable Metadata API](https://github.com/uc-cdis/metadata-service#metadata-service) and programmatically access semi-structured metadata using the [endpoint](https://petstore.swagger.io/?url=https://raw.githubusercontent.com/uc-cdis/metadata-service/master/docs/openapi.yaml)

Reply:
Updated.  Thanks for the suggestion.